constrained-voucher.mkd

```diff
@@ -118,25 +118,20 @@ Rather than serializing the YANG definition in JSON, it is serialized into CBOR
 
 This document follows a similar, but not identical structure as {{RFC8366}} and supplements the brski part to {{I-D.ietf-ace-coap-est}}.
 
-There are three constrained situations described in this document:
-1. CMS signed CBOR encoded vouchers transported using CoAP, protected by DTLS (coaps).
-2. COSE signed CBOR encoded vouchers transported using CoAP, protected by EDHOC or DTLS.
-3. COSE signed CBOR encoded vouchers, integrated into the key exchange as described by {{!I-D.selander-ace-ake-authz}}
+This document describes a single new voucher format.
+It is a COSE-signed CBOR-encoded vouchers transported using CoAP.
```

Comment:
vouchers -> voucher
the transport can also be HTTPS, from MASA to Registrar; not only CoAP.